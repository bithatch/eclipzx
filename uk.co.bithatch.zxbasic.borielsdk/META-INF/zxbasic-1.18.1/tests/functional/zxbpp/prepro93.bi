
REM Checks -D MACRO=VALUE is defined correctly from zxbc commandline

#ifdef MACRO
#  if MACRO != VALUE
#    error This should not happen
#  endif
#else
OK
#endif
