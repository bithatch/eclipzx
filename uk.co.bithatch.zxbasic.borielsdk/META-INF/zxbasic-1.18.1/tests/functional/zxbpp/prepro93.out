#line 1 "prepro93.bi"



#line 9 "prepro93.bi"
OK
#line 11 "prepro93.bi"
